tensorflow/tools/ci_build/builds/pip_test.sh

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env bash
```

Comment:
Nice. It is good to pull out the test only script. No compilation or installation here.

Should it be called test_installation.sh? Or something like that? I can be used to run python test on tf installed anywhere by any means.